### PR TITLE
+ Added a file input form control

### PR DESCRIPTION
**Pull request by sroy3**
screen shot 2016-08-29 at 4 19 57 pm

screen shot 2016-08-29 at 4 20 20 pm

### CODE REVIEW

File: scss/controls/file-input.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    input {
+      position: absolute;
+      top: 0;
+      left: 0;
+
+      width: 100%;
+      height: 100%;
+
+      cursor: pointer;
+      opacity: 0;
```
Comment (wfortin) [inline scss/controls/file-input.scss]: Is position absolute really needed since we are in a flex container ?
Cant we juste use `align-self: flex-start` ?

Comment (sroy3) [inline scss/controls/file-input.scss]: It's the hidden file input so that wherever we click, we open the choose file window.

Comment (wfortin) [inline scss/controls/file-input.scss]: Ha makes more sense.

**Comment by wfortin** [comment]
🚢 it

**Comment by pochretien** [comment]
🚢 🇮🇹

**Pull request merged**